server/controllers/finance/vouchers.js

```diff
@@ -10,7 +10,10 @@
 * @requires node-uuid
 * @requires lib/util
 * @requires lib/db
+* @requires lib/ReportManager
 * @requires lib/errors/NotFound
+* @requires lib/errors/BadRequest
```

Comment:
:+1:  Thanks for adding these additional `@requires`.  It really helps add polish to the code and documentation!